btr-web/btr-common-components/utils/date.ts

```diff
@@ -6,6 +6,8 @@ import moment from 'moment'
 export function dateStringToDate (dateString: string) {
   // convert to date
   const date = new Date(dateString)
+  // @ts-ignore
+  if (isNaN(date)) { return null }
```

Comment:
are you sure that this will work ?
Is JS treating date as number ?

Reply:
yeah I read about this online and then tested it -- catches invalid dates (i.e. if I called this function with 'qwe')